account_invoice_inter_company/views/res_config_settings_view.xml

```diff
@@ -51,6 +51,20 @@
                         </div>
                     </div>
                 </div>
+                    <div class="col-12 col-lg-6 o_setting_box" id="invoice_queued">
```

Comment:
Don't add this here, but on the new module. Name the div or group you need for hooking it correctly though.